--- a/setup.py  2024-12-26 21:19:27.000000000 +0530
+++ b/setup.py  2025-11-15 04:16:49.174749661 +0530
@@ -152,8 +152,8 @@
 platform = sys.platform

 # Detect Python for android project (http://github.com/kivy/python-for-android)
-ndkplatform = environ.get('NDKPLATFORM')
-if ndkplatform is not None and environ.get('LIBLINK'):
+ndkplatform = environ.get('NDK_ROOT')
+if ndkplatform is not None:
     platform = 'android'
 kivy_ios_root = environ.get('KIVYIOSROOT', None)
 if kivy_ios_root is not None:

--- a/kivy/include/gl_redirect.h
+++ b/kivy/include/gl_redirect.h
@@ -56,7 +56,7 @@
 
 #else
 
-#	if __USE_OPENGL_ES2
+#	if 1
 #		if __APPLE__
 #			include "common_subset.h"
 #		else
